package.json

```diff
@@ -27,6 +27,7 @@
     }
   },
   "scripts": {
+    "bootstrap": "lerna bootstrap",
```

Comment:
Why do we need this? One can simply call `lerna bootstrap` from the terminal. Please remove this script again. Also, we have the `make clean` command available which additionally cleans up before.